src/tools/cephfs/cephfs-shell

```diff
@@ -1539,8 +1540,14 @@ if __name__ == '__main__':
     main_parser.add_argument('-t', '--test', action='store',
                              help='Test against transcript(s) in FILE',
                              nargs='+')
-    main_parser.add_argument('commands', nargs='*',
-                             help='comma delimited commands', default=[])
+    main_parser.add_argument(
+        'commands',
+        nargs='*',
+        help='Comma delimited commands. The shell executes the given command '
+             'and quits immediately with the return value of command. In '
+             'case no commands are provided, the shell is launched.',
```

Comment:
```suggestion
        help='The shell executes the given command  and quits immediately '
             'with the return value of command. In case no commands are'
             'provided, the shell is launched. Comma delimited commands. ',
```

Reply:
Why move the brief description (`Comma delimited commands.`) to the end? Keeping it at the beginning allows users a quick glance.

Reply:
It is fine if you want to keep it this way.